dataikuapi/dssclient.py

```diff
@@ -743,6 +743,19 @@ def log_custom_audit(self, custom_type, custom_params=None):
             body = custom_params)
 
     ########################################################
+    # Monitoring
+    ########################################################
+
+    def get_global_usage_summary(self):
```

Comment:
From Marc's comment (https://github.com/dataiku/dip/pull/13032#discussion_r672957865), having an option to request the metrics per project would be cool